tensorflow_addons/optimizers/moving_average_test.py

```diff
@@ -107,25 +107,30 @@ def test_model_weights_update(self):
         self.evaluate(mean_update)
         self.assertAllClose(model.variables[0].read_value(), [[0.9]])
 
+    def test_optimizer_string(self):
+        _ = MovingAverage('adam')
```

Comment:
@Squadrick  Hi, Dheeraj, do we forget to check the result in the case?

Reply:
@facaiy No, just needed to ensure that creating the optimizer with a string doesn't throw any errors.